starknet_py/cairo/serialization/serialized_payload.py

```diff
@@ -0,0 +1,15 @@
+from collections import OrderedDict
+
+from starknet_py.cairo.felt import CairoData
+
+
+class SerializedPayload(OrderedDict[str, CairoData]):
+    """
+    Dictionary for serialized payload members. Use ``to_calldata`` to get raw calldata.
+    """
+
+    def to_calldata(self) -> CairoData:
+        result = []
+        for entry_calldata in self.values():
+            result.extend(entry_calldata)
+        return result
```

Comment:
```suggestion
        return [result for result in self.values()]
```
I think it's more concise.

Reply:
The suggestion is not equivalent, is it?
Originally there was `extend` method used, whereas the suggestion is the same as `append`.

Reply:
Hmm, so `entry_calldata` can be a list instead of a single item? Only then it should matter.

Reply:
The class inherits from `OrderedDict[str, CairoData]` where `CairoData = List[int]`.

Reply:
We can turn it into flatmap list comprehension, but it might be less readable.

Reply:
If `extend` is necessary, I think it's okay to keep it the way it currently is.